DigitalLearningSolutions.Data/DataServices/UserDataService/AdminUserDataService.cs

```diff
@@ -123,6 +123,21 @@ ELSE cc.CategoryName
             ).SingleOrDefault();
         }
 
+        public IEnumerable<AdminEntity> GetAdminsByCentreId(int centreId)
+        {
+            var sql = $@"{BaseAdminEntitySelectQuery} WHERE aa.centreID = @centreId";

```

Comment:
Why doesn't this query check that the user/admin account is active (like `GetAdminUsersByCentreId` does)? The scope of the ticket doesn't appear to require the display of inactive admin accounts. I think `aa.Active = 1` is appropriate, and possibly `u.Active = 1` as well